workshops/Tkinter/README.md

```diff
@@ -0,0 +1,188 @@
+---
+name: "To Quote or Not To Quote?"
+description: "Let's build a slideshow of quotes with Tkinter and Python!"
+author: "@bezlin6mechminerz"
+---
+
+The workshop will look something like this.
+
+![ezgif.com-video-to-gif-3.gif](https://cloud-beibqb64y.vercel.app/ezgif.com-video-to-gif-3.gif)
+
+So we will be making this software. A quotes hub which displays quotes.
+
+View a [live_demo](https://repl.it/@bezlin/graphics#main.py)
+
+View the [final_code](https://repl.it/@bezlin/graphics#main.py)
+
+This workshop will take 20 minutes.
+
+## Getting started
+
+We will be using [Tkinter](https://docs.python.org/3/library/tk.html) python library to develop the user interface.
+
+### Setting up the environment for Tkinter
+
+[Repl.it](https://repl.it) We will be using it for coding. This is awesome because you no need to install python on your machine. Just go to this link.
+
+To get started, go to https://repl.it/languages/Tkinter. Your coding environment will be ready in a few seconds!
+<img width="1440" alt="Screenshot 2020-09-18 at 5 39 06 PM" src="https://cloud-o8q46nkko.vercel.app/screenshot_2020-09-25_at_9.23.06_pm.png">
+
+## What is Tkinter
+
+Python offers multiple options for developing a GUI (Graphical User Interface). Out of all the GUI methods, Tkinter is the most commonly used method. It is used to make software and Graphical User interfaces for your python code.
```

Comment:
maybe move this up to be after "We will be using [Tkinter](https://docs.python.org/3/library/tk.html) python library to develop the user interface."